app/routers/event.py

```diff
@@ -84,6 +90,60 @@ async def eventedit(request: Request) -> Response:
     return templates.TemplateResponse("eventedit.html", {"request": request})
 
 
+@router.get("/edit/view_countries", include_in_schema=False)
```

Comment:
`/edit/view` might be confusing.
If this page shows the name of the countries, call it `/view/countries`.
Maybe even create it under a different router, because it's not countries of a specific event, so the full URL should look like `calendar.pythonic.guru/countries`